static/usr/share/subiquity/console-conf-write-login-details

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+
+pretty_name="$( awk -F "=" '$1 = /PRETTY_NAME/ {print $2;}' /usr/lib/os-release | sed 's/"//g')"
+tty_name=$(tty | cut -c 6-)
+
+get_first_ip() {
+  for iface in $(ip -br address | awk '{print $1}')
+  do
+    if test -n ${iface} && ip addr show dev ${iface} >/dev/null 2>&1; then
+      if test -e /sys/class/net/${iface}/device; then
+        local ip_address="$(ip -4 -br addr show dev ${iface} | awk '{print$3}' | sed 's|\(.*\)/.*|\1|')"
+        if [ -n "${ip_address}" ]; then
+          echo "${ip_address}"
+          return 0
+        fi
+      fi
+    fi
+  done
+  echo "<no ip address>"
+  return 2
+}
+
+print_ssh_commands() {
+  # take any user with non zero
+  for h in /root $(ls /home/ | awk '{print "/home/"$1}')
+  do
+    # ignore users without .ssh/authorized_keys
+    if test ! -e ${h}/.ssh/authorized_keys || test "0" = "$( stat ${h}/.ssh/authorized_keys | awk '$1 == "Size:" {print $2}')"; then
+      continue
+    fi
+    local user_name="$(basename ${h})"
+    for iface in $(ip -br address | awk '$2 == "UP" {print $1}')
+    do
+      if test -n ${iface} && ip addr show dev ${iface} >/dev/null 2>&1; then
+        if test -e /sys/class/net/${iface}/device; then
```

Comment:
Why filtering out bridge interfaces? If the device is configured with bridge, then the hardware interfaces will have no ip addresses. And here we would get nothing. I would just show all IP from all interfaces that have an address.

Reply:
I think this was done by accident in the original code, when the idea was to skip `lo`. I've dropped this and added explicit check for `lo`.